### DISCUSSION TITLE
Datasets are being used for spam

### DISCUSSION THREAD

**Discussion by mandrewj** [Symbiota/community-board#175] [Problems/Bugs]
Similar to the former checklist spam posts that I believe were patched with a recent security update, publicly viewable datasets are also being created.

As an example, which I will delete soon, see this on ecdysis:
[399705](https://serv.biokic.asu.edu/ecdysis/collections/datasets/public.php?datasetid=19)

Happy to post this on issues instead if it isn't already an internally known issue, though I figured this might be better posted here first?

Not sure the best workaround since I really really really like the idea of everyone being able to make datasets - perhaps some moderation or permission tool before making them public?

**Comment by ljwalker** [comment]
Hi Andrew! We're currently working on it. We'll be sending a communication about it to the community once we finish cleaning everything up.

**Comment by GregoryPost** [comment]
Hi Andrew,

The dataset spam issue was patched at the same time as the checklist and relies on the same new permission.  Going forward, users will not be able to create a checklist or a dataset unless they are granted the "Create a Checklist" role.  

![image](https://user-images.githubusercontent.com/98365364/176471441-fc0cd2d6-b892-4ca4-8fa1-ba5471576cf1.png)

Note that this role/permission is not a prerequisite for granting a user admin permission on a specific checklist.

I have deleted the offending dataset and the associated user from ecdysis.

-Greg
